lib/vc.js

```diff
@@ -35,18 +35,22 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 'use strict';
+
 const jsonld = require('jsonld');
 const jsigs = require('jsonld-signatures');
 const CredentialIssuancePurpose = require('./CredentialIssuancePurpose');
-const defaultDocumentLoader = require('./documentLoader');
+const defaultDocumentLoader = jsigs.extendContextLoader(
+  require('./documentLoader'));
 
 module.exports = {
   issue,
   verify,
+  createPresentation,
   verifyPresentation,
```

Comment:
I recommend we don't even expose this externally. Make people use `verify` so there's just one way to do it.

```suggestion
```

Reply:
It's exported for ease of unit testing.

Reply:
But we can unit test using `verify`? If we still need it exposed then please prefix an underscore instead to help indicate it's private and can't be relied upon.

Reply:
Np, renamed to `_verifiedPresentation()`